planet/cli/data.py

```diff
@@ -295,6 +342,32 @@ async def search_create(ctx, name, item_types, filter, daily_email, pretty):
         echo_json(items, pretty)
 
 
+@data.command()
+@click.pass_context
+@translate_exceptions
+@coro
+@click.argument("item_types", callback=parse_item_types)
```

Comment:
you can remove `parse_item_types` and just use `types.CommaSeparatedString`, see [search-create](https://github.com/planetlabs/planet-client-python/blob/f376018877daf845d3666359082475a25baf9f47/planet/cli/data.py#L271) for an example.